source/plugins/code/README.md

```diff
@@ -0,0 +1,27 @@
+### ♐ Code snippet of the day
+
+> ⚠️ When improperly configured, this plugin could display private code. If you work with sensitive data or company code, it is advised to keep this plugin disabled. *Metrics* and its authors cannot be held responsible for any eventual code leaks, use at your own risk.
```

Comment:
```suggestion
> ⚠️ When improperly configured, this plugin could display private code. If you work with sensitive data or company code, it is advised to keep this plugin disabled. *Metrics* and its authors cannot be held responsible for any code leaks, use at your own risk.
```
I think "eventual" implies that the code leak is guaranteed to happen at some point, so it might be best to exclude it. Alternatively, maybe something like "resulting".

Reply:
Thanks, I forgot that "eventual" had this meaning in English 🙂